dependencies/adi/command_parser/command_parser.cpp

```diff
@@ -24,15 +24,11 @@
 
 #include "command_parser.h"
 #include <iostream>
+#include <map>
```

Comment:
You already included <map> in command_parser.h

Reply:
Solved